Makefile

```diff
@@ -19,8 +24,11 @@ test:
 
 .PHONY: test-docker test-container
 test-docker: test-container
-test-container: check-ceph-version .build-docker
-	$(CONTAINER_CMD) run --device /dev/fuse --cap-add SYS_ADMIN $(CONTAINER_OPTS) --rm -v $(CURDIR):/go/src/github.com/ceph/go-ceph$(VOLUME_FLAGS) $(DOCKER_CI_IMAGE)
+test-container: check-ceph-version .build-docker $(RESULTS_DIR)
+	$(CONTAINER_CMD) run --device /dev/fuse --cap-add SYS_ADMIN $(CONTAINER_OPTS) --rm -v $(CURDIR):/go/src/github.com/ceph/go-ceph$(VOLUME_FLAGS) $(RESULTS_VOLUME) $(DOCKER_CI_IMAGE)
+
```

Comment:
I would probably put this new target inside a `ifdef RESULTS_DIR` statement. It may not be required now, but I expect it would be when there are other optional/empty targets.

Reply:
I did that. Leaving PR open for a bit in case you have addtional feedback... but will merge later since you approved already